scripts/competitiond.py

```diff
@@ -708,7 +708,9 @@ def generate_leaderboard(self, num_total_submissions, num_period_submissions):
 
         # Sort by the scores, descending
         leaderboard.sort(
-            key=lambda e: tuple(e['scores'][spec['name']] for spec in self.config['score_specs']),
+            key=lambda e: tuple(
+                e['scores'][spec['name']] is not None for spec in self.config['score_specs']
```

Comment:
Maybe this would fix it @candicegjing ?

```suggestion
                e['scores'][spec['name']] or 0 for spec in self.config['score_specs']
```

Reply:
I think with `is not None`, we sort all the bundles with `null` scores at the end of the result and all the `not null` scores on the top.